examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2105,623 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
+				{"rating", 4.3},
+			},
+			bson.D{
+				{"name", "Dad's Grill"},
+				{"city", "Oklahoma City"},
+				{"state", "Oklahoma"},
+				{"country", "United States"},
+				{"rating", 2.1},
+			},
+		}
+
+		result, err := coll.InsertMany(ctx, restaurants)
+
+		// End RunCommand Example 1
+
+		require.NoError(t, err)
+		require.Len(t, result.InsertedIDs, 5)
+	}
+	{
+		// Start RunCommand Example 2
+		res := db.RunCommand(ctx, bson.D{{"buildInfo", 1}})
+
+		// End RunCommand Example 2
+
+		err := res.Err()
+		require.NoError(t, err)
+	}
+	{
+		// Start RunCommand Example 3
+		res := db.RunCommand(ctx, bson.D{{"collStats", "restaurants"}})
+
+		// End RunCommand Example 3
+
+		err := res.Err()
+		require.NoError(t, err)
+	}
+}
+
+// IndexExamples contains examples of Index operations.
+func IndexExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	recordsColl := db.Collection("records")
+	restaurantsColl := db.Collection("restaurants")
+
+	err := recordsColl.Drop(ctx)
+	require.NoError(t, err)
+	err = restaurantsColl.Drop(ctx)
+	require.NoError(t, err)
+
+	{
+		// Start Index Example 1
+		records := []interface{}{
+			bson.D{
+				{"student", "Marty McFly"},
+				{"classYear", 1986},
+				{"school", "Hill Valley High"},
+				{"GPA", 2.1},
+			},
+			bson.D{
+				{"student", "Ferris F. Bueller"},
+				{"classYear", 1987},
+				{"school", "Glenbrook North High"},
+				{"status", "Suspended"},
+				{"GPA", 3.4},
+			},
+			bson.D{
+				{"student", "Reynard Muldoon"},
+				{"classYear", 2007},
+				{"school", "Stonetown Middle"},
+				{"GPA", 4.0},
+			},
+		}
+		restaurants := []interface{}{
+			bson.D{
+				{"name", "Chez Panisse"},
+				{"cuisine", "American/French"},
+				{"city", "Oakland"},
+				{"state", "California"},
+				{"country", "United States"},
+				{"rating", 4.9},
+			},
+			bson.D{
+				{"name", "Central"},
+				{"cuisine", "Peruvian"},
+				{"city", "Lima"},
+				{"country", "Peru"},
+				{"rating", 5.8},
+			},
+			bson.D{
+				{"name", "Eleven Madison Park"},
+				{"cuisine", "French"},
+				{"city", "New York City"},
+				{"state", "New York"},
+				{"country", "United States"},
+				{"rating", 7.1},
+			},
+			bson.D{
+				{"name", "Gaggan"},
+				{"cuisine", "Thai Fusion"},
+				{"city", "Bangkok"},
+				{"country", "Thailand"},
+				{"rating", 9.2},
+			},
+			bson.D{
+				{"name", "Dad's Grill"},
+				{"cuisine", "BBQ"},
+				{"city", "Oklahoma City"},
+				{"state", "Oklahoma"},
+				{"country", "United States"},
+				{"rating", 2.1},
+			},
+		}
+
+		recordsResult, recordsErr := recordsColl.InsertMany(ctx, records)
+		restaurantsResult, restaurantsErr := restaurantsColl.InsertMany(ctx, restaurants)
+
+		// End Index Example 1
+
+		require.NoError(t, recordsErr)
+		require.Len(t, recordsResult.InsertedIDs, 3)
+		require.NoError(t, restaurantsErr)
+		require.Len(t, restaurantsResult.InsertedIDs, 5)
+	}
+	{
+		// Start Index Example 3
+		indexModel := mongo.IndexModel{
+			Keys: bson.D{
+				{"GPA", 1},
```

Comment:
From the ticket, i think this should be example 1 and GPA should be score

Reply:
Switched to `Index Example 2` (see comment above about the example offset), and `score`.